Validation: description-missing fixes. (#239)
MicrosoftDocs/bingmaps-docs: BingMaps/v8-web-control/modules/autosuggest-module/autosuggestoptions-object.md

---
title: "AutosuggestOptions Object | Microsoft Docs"

> description: Describes the AutosuggestOptions object, which controls autosuggest options, and provides a list of the object's properties.

Reviewer: ```suggestion
description: Describes the AutosuggestOptions object, which controls autosuggest options, and descriptions of each of its properties.
```
